channeldb/payment_control.go

```diff
@@ -95,12 +95,20 @@ type PaymentControl struct {
 	currPaymentSeq   uint64
 	storedPaymentSeq uint64
 	db               *DB
+	cfg              *PaymentControlConfig
+}
+
+// PaymentControlConfig houses necessary fields that a PaymentControl needs to
+// function.
+type PaymentControlConfig struct {
+	KeepFailedPayments bool
```

Comment:
nit: add godoc since this member is exported.

Reply:
Done.